### PR TITLE
Support for AEG Devices & de.json

### PR DESCRIPTION
**Pull request by MickLesk**
Added de.json

Added support: AX5, AX7, AX9 Devices (AEG)

### CODE REVIEW

File: custom_components/wellbeing/api.py [hunk 1]
```diff
@@ -240,14 +246,24 @@ def setup(self, data):
 
     @property
     def speed_range(self) -> tuple:
+        ## Electrolux Devices: 
+        if self.model == "WELLA5":
+            return 1, 5
         if self.model == "WELLA7":
             return 1, 5
         if self.model == "AX7":
```
Comment (JohNan) [inline custom_components/wellbeing/api.py:254]: Duplicate. Please remove it.

**Review by JohNan — CHANGES_REQUESTED** [review]

File: custom_components/wellbeing/api.py (hunk 1 above)
Comment (MickLesk) [reply custom_components/wellbeing/api.py:254]: where is the duplicate? 

the current code of your branch:
image

Comment (JohNan) [reply custom_components/wellbeing/api.py:254]: See your diff https://github.com/JohNan/homeassistant-wellbeing/blob/1868cbb0f6eb45df24377ee9809d0e2420346118/custom_components/wellbeing/api.py#L254-L263

**Review by JohNan — APPROVED** [review]
Thank you!

**Pull request merged**